src/Frontend/state/actions/popup-actions/popup-actions.ts

```diff
@@ -416,3 +487,13 @@ export function locateSignalsFromProjectStatisticsPopup(
     }
   };
 }
+
+export function removeWasPreferred(): AppThunkAction {
+  return (dispatch: AppThunkDispatch, getState: () => State): void => {
+    const temporaryDisplayPackageInfo = getTemporaryDisplayPackageInfo(
```

Comment:
if you do this, you won't need the `delete`:
```suggestion
    const { wasPreferred, ...temporaryDisplayPackageInfo } = getTemporaryDisplayPackageInfo(
```

Reply:
Neat, I didn't know about that.